_commands/basics/sleep.md

```diff
@@ -3,22 +3,37 @@
 
 sleep
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+The sleep command makes your script, terminal, or anything else that is bash executable go to sleep for however long you specify 
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+sleep 5s
```

Comment:
Also another convention, include a `$ ` before the command itself if you're showing example usage, i.e. `$ sleep 5s`. You'd need to change this a couple times throughout the file